generator/config.go

```diff
@@ -16,9 +16,9 @@ type Config struct {
 	// Option specify options to be passed gRPC Federation code generator.
 	Option *PluginOption `yaml:"opt"`
 	// AutoProtocGenGo automatically run protoc-gen-go at the time of editing proto. default is true.
-	AutoProtocGenGo *bool `yaml:"auto-protoc-gen-go"`
+	AutoProtocGenGo *bool `yaml:"autoProtocGenGo"`
 	// AutoProtocGenGoGRPC automatically run protoc-gen-go-grpc at the time of editing proto. default is true.
-	AutoProtocGenGoGRPC *bool `yaml:"auto-protoc-gen-go-grpc"`
```

Comment:
The linter raises an error to a snake-cased tag, but if it is intentional, I'll return it.